### DISCUSSION TITLE
How to get crop_center in RandSpatialCropd

### DISCUSSION THREAD

**Discussion by XIANYUNYEHE-DEL** [Project-MONAI/MONAI#5716] [Q&A] [answered]
I want to get crop location after transform, but only in `RandCropByPosNegLabel` and `RandCropByLabelClasses`  I can get crop_center. How do I get this information in `RandSpatialCropd` or `ResizeWithPadOrCrop`

**Comment by wyli** [comment, marked as answer]
the cropping margins are stored in `result.applied_operations`
something like:
```
[{class: 'RandSpatialCrop', id: 140677504472448, orig_size: torch.Size([4, 5, 6]), extra_info: {'cropped': [0, 0, 0, 1, 0, 3]}}]
```

**Reply by XIANYUNYEHE-DEL** [reply]
but my `applied_operations` is none.
```
loader = Dataset(data=data_dicts,transform=train_transform)
train_loader = DataLoader(loader,
                        batch_size=2,
                        shuffle=False,
                        num_workers=8,
                        pin_memory=True,
                        persistent_workers=True)
for idx, batch_data in enumerate(train_loader):
    data, target1 = batch_data['image'], batch_data['label'] 
    print(data.applied_operations)
```

**Reply by wyli** [reply]
I've checked that this example works fine though https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/torch/unet_training_array.py
